File: problem_1/Ruby/p1.rb
@@num = 0
i = 1

while ( i < 1000 )
  if i % 3 == 0 then
    @@num += i
  elsif i % 5 == 0 then
    @@num += i
  end
  i += 1
end

printf("answer => %d\n" , @@num)

File: problem_1/Go/p1.go
package main

import fmt "fmt"
func main() {
     num:= 0
     for i:= 1; i < 1000; i++ {
         if i % 3 == 0 {
            num += i
         } else if i % 5 == 0 {
            num += i
         }
     }
     fmt.Printf("answer => %d", num)

}